core/ExposedTest.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "outputs": [],
+   "source": [
+    "USE { dependencies(\"org.xerial:sqlite-jdbc:3.30.1\") }"
+   ],
+   "metadata": {
```

Comment:
Could you please move such notebooks like this or spark to "research branch or as a separate PR" when you finish or probably to your own repository with dependency to our KDF? It will be quite interesting!

Reply:
oops these should not have been added. But yeah that sounds like a good idea :)